src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
+import { TypeMovieList } from '../../types';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/index';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" key={movie.imdbId}>
+        <MovieCard {...movie} />
+      </div>
+    ))}
```

Comment:
```suggestion
      <MovieCard  key={movie.imdbId} {...movie} />
    ))}
```
You can add div with class 'card' inside of your MovieCard component, as it's a part of the card